.github/workflows/workflow-test-otelcol-config.yml

```diff
@@ -0,0 +1,77 @@
+name: Run tests (otelcol-config)
+
+on:
+  workflow_call:
+    inputs:
+      arch_os:
+        description: Architecture and OS in the form "{arch}_{os}". See GOARCH and GOOS for accepted values.
+        default: linux_amd64
+        type: string
+      boringcrypto:
+        description: Run with BoringCrypto enabled
+        default: false
+        type: boolean
+      runs-on:
+        default: ubuntu-20.04
+        type: string
+      only-if-changed:
+        description: Run only if relevant files changed.
+        default: false
+        type: boolean
+      save-cache:
+        description: Save the module and build caches.
+        default: false
+        type: boolean
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GO_VERSION: "1.21.4"
```

Comment:
This should be upgraded to at least 1.21.11

Reply:
Done!